R/stat-boxplot.r

```diff
@@ -45,6 +45,11 @@ StatBoxplot <- ggproto("StatBoxplot", Stat,
   {
     data <- remove_missing(data, na.rm, c("x", "y", "weight"), name="stat_boxplot",
       finite = TRUE)
+    if (!is.discrete(data$x) && data$group[1L] == NO_GROUP && any(data$x != data$x[1L])) {
```

Comment:
Thanks. I'll add a test later tonight.

`data$group[1L] == NO_GROUP` is much faster for the (expected) case that the user has specified a group. I'd rather not add an extra slowdown, instead I could add a comment or refactor this into a function. Please confirm.